client/capital/index.tsx

```diff
@@ -223,10 +229,12 @@ const CapitalPage = (): JSX.Element => {
 				title={ __( 'All loans', 'woocommerce-payments' ) }
 				isLoading={ isLoading }
 				totalRows={ loans.length }
-				headers={ columns }
+				headers={ columnsToDisplay }
+				onColumnsChange={ onColumnsChange }
 				rows={ getRowsData( loans ) }
 				rowsPerPage={ loans.length }
 				summary={ getSummary( loans ) }
+				// TODO: Confirm if we need to show the menu to allow users to hide columns
```

Comment:
💭 Since the three-dot menu is disabled on the Capital Loan table, the visibility of columns cannot be modified and therefore.

We should either enable the menu or remove the persistence hook.

Reply:
Yes, to see the visibility toggle, we'd also need to set one or more columns to `required: false`.

https://github.com/Automattic/woocommerce-payments/blob/4766038db715fa9769027980e1a8daaff2a09382/client/capital/index.tsx#L51

Reply:
I've opened #10106 to handle this and I'll remove the `TODO` comment.